### PR TITLE
Fix regex pattern

### PR DESCRIPTION
**Pull request by kunalgupta022**
Fix the regex pattern for replacing the filepath from the parsed diff

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ async function getChangedFilesPaths(token) {
   const parsedDiff = gitDiffParser(pullRequestDiff);
 
   const changedFilePaths = parsedDiff.map((e) => {
-    return e['newPath'].replace('b/', '');
+    return e['newPath'].replace(/^b\//, '');
```
Comment (gps) [inline dist/index.js:57]: Why is this needed in the first place?

Comment (kunalgupta022) [reply dist/index.js:57]: The git diff parser outputs the file paths in the following format:
1. Files that are modified: `{'oldPath': 'a/path/to/file', 'newPath': 'b/path/to/file'}`
2. Files that are remaned or added`{'oldPath': null, 'newPath': 'b/path/to/file'}`

Hence the need to remove the first occurrence of `b/` from the file path.

Comment (kunalgupta022) [reply dist/index.js:57]: Actually, the `b/` is supposed to be there for all `newPath` values according to my tests but for some reason it probably was not there for the specific case that you sent me with the bug which is odd.

**Review by gps — APPROVED** [review]
